packages/ui/Layout/Cell.js

```diff
@@ -70,6 +70,21 @@ const CellBase = kind({
 		 */
 		componentRef: EnactPropTypes.ref,
 
+		/**
+		 * Sizes `Cell` to its container.
+		 *
+		 * A `grow`able cell will expand to its maximum size, according to the remaining space of the
+		 * container. This is used when you want to grow the size of this Cell so that it fills the
+		 * container. See the {@link ui/Layout.Cell#size|size} property for more details.
+		 *
+		 * When combined with {@link ui/Layout.Cell#shrink|shrink}, `shrink` prop takes precedence over
+		 * `grow` prop and `grow` prop is simply ignored.
+		 *
+		 * @type {Boolean}
+		 * @public
+		 */
+		grow: PropTypes.bool,
```

Comment:
Would you please add unit tests for this prop?

Reply:
OK. I will add unit tests for this prop.